geode-redis/src/acceptanceTest/java/org/apache/geode/redis/internal/executor/server/InfoNativeRedisAcceptanceTest.java

```diff
@@ -28,9 +28,4 @@
   public int getPort() {
     return redis.getPort();
   }
-
-  @Override
-  int getExposedPort() {
```

Comment:
You will probably need to revert this. When run under docker, there are 2 ports in play - one is the external port exposed for outside access and one is the internal port which the external port is proxied to. It's the internal port that this is returning and that's the port that the Info command will report.